src/Common/src/System/IO/DelegatingStream.cs

```diff
@@ -172,6 +172,11 @@ public override void EndWrite(IAsyncResult asyncResult)
             _innerStream.EndWrite(asyncResult);
         }
 
+        public override void CopyTo(Stream destination, int bufferSize)
+        {
+            _innerStream.CopyTo(destination, bufferSize);
+        }
```

Comment:
It shouldn't be an #if but rather a partial class file with a .netcoreapp.cs extension.